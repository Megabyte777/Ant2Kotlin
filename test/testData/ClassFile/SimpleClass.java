package testData.ClassFile;

public class SimpleClass {
    public static void method1() {}

    public int method2() {
        return 0;
    }

    private String method3() {
        return null;
    }

    protected void method4() {}
}
