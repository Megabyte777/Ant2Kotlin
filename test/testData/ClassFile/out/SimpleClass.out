method4()V
method1()V
<init>()V
method2()I
method3()Ljava/lang/String;
